### PR TITLE
Add APDU to get extended public address

### PR DESCRIPTION
**Pull request by dfordivam**

### CODE REVIEW

File: src/apdu.h [hunk 1]
```diff
@@ -28,7 +28,7 @@
 #define INS_GET_WALLET_ID             0x01
 #define INS_PROMPT_PUBLIC_KEY         0x02
 #define INS_SIGN                      0x03
-#define INS_SIGN_UNSAFE               0x05 // Data that is already hashed.
+#define INS_PROMPT_EXT_PUBLIC_KEY     0x04
 #define INS_RESET                     0x06
 #define INS_QUERY_AUTH_KEY            0x07
```
Comment (jhartzell42) [inline src/apdu.h:33]: Please feel free to clear out dead Tezos code as you find it.

File: src/apdu_pubkey.c [hunk 2]
```diff
@@ -98,17 +120,22 @@ size_t handle_apdu_get_public_key(uint8_t _U_ instruction) {
...
 
     // write tags
     GPriv.prefixed_public_key_hash.address_type_is_short = 0x01;
     GPriv.prefixed_public_key_hash.key_hash_type_is_sighash = 0x00;
 
     // write lock arg
-    generate_lock_arg_for_pubkey(&G.public_key, &GPriv.prefixed_public_key_hash.hash);
+    generate_lock_arg_for_pubkey(&G.ext_public_key.public_key, &GPriv.prefixed_public_key_hash.hash);
 
     // instruction == INS_PROMPT_PUBLIC_KEY || instruction == INS_AUTHORIZE_BAKING
```
Comment (jhartzell42) [inline src/apdu_pubkey.c:132]: This is also a stale Tezos comment, please remove these if close to where you are working

File: src/apdu_pubkey.c [hunk 3]
```diff
@@ -69,6 +74,23 @@ void render_pkh(char *const out, size_t const out_size,
 }
 
 __attribute__((noreturn)) static void prompt_path(ui_callback_t ok_cb, ui_callback_t cxl_cb) {
+  static size_t const TYPE_INDEX = 0;
+  static size_t const DRV_PATH_INDEX = 1;
+  static size_t const ADDRESS_INDEX = 2;
+
+  static const char *const pubkey_labels[] = {
+                                              PROMPT("Provide"),
+                                              PROMPT("Derivation Path"),
```
Comment (jhartzell42) [inline src/apdu_pubkey.c]: This isn't the prompt from the document.

Comment (dfordivam) [reply src/apdu_pubkey.c]: I have added #80 for the prompt

**Pull request merged**